docs/sources/flow/reference/components/beyla.ebpf.md

```diff
@@ -0,0 +1,250 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/beyla.ebpf/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/beyla.ebpf/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/beyla.ebpf/
+- /docs/grafana-cloud/send-data/agent/flow/reference/components/beyla.ebpf/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/beyla.ebpf/
+description: Learn about beyla.ebpf
+title: beyla.ebpf
+---
+
+# beyla.ebpf
```

Comment:
Shall we mention that this component might be some versions behind the `latest` version of Beyla?

Reply:
I think is usually not necesary specify the version of that a component uses. @ptodev do you have any insights here? We probably should mark as beta, what do you think?

Reply:
I'm thinking in the time between we publish a new Beyla version with new public features and we later update the Beyla library here (and publish a new version of the Agent)... maybe a user would expect that a given feature is working already in the Agent.